src/components/screens/monitor/blocks/blocks.js

```diff
@@ -20,76 +25,106 @@ const columnClassNames = {
+      ...acc,
+      ...(filters[key] && { [key]: filters[key] }),
+    }), {
+      offset: blocks.data.length,
+    }));
+  };
+
+  return (
+    <PageLayout>
+      <Box isLoading={blocks.isLoading} width="full" main>
+        <Box.Header>
+          <h1>{t('All blocks')}</h1>
+          <BlockFilterDropdown filters={filters} applyFilters={applyFilters} />
+        </Box.Header>
+        <FilterBar {...{
+          clearFilter, clearAllFilters, filters, formatters, t,
+        }}
+        />
+        { blocks.error
+          ? (
+            <Box.Content>
+              <Feedback status="error" show>{`${blocks.error}`}</Feedback>
+            </Box.Content>
+          )
+          : (
+            <React.Fragment>
+              <Box.Content className={styles.content}>
+                {!!blocks.data.length && (
+                <TableRow isHeader className={`${grid.row}`}>
+                  <div className={columnClassNames.id}>{t('ID')}</div>
+                  <div className={columnClassNames.height}>{t('Height')}</div>
+                  <div className={columnClassNames.date}>{t('Date')}</div>
+                  <div className={columnClassNames.transactions}>{t('Transactions')}</div>
+                  <div className={columnClassNames.generator}>{t('Generated by')}</div>
+                  <div className={columnClassNames.amount}>{t('Amount')}</div>
+                  <div className={columnClassNames.forged}>{t('Forged')}</div>
+                </TableRow>
+                )}
+                {blocks.data.map(block => (
+                  <TableRow key={block.id} className={`${grid.row}`}>
+                    <span className={[columnClassNames.id, 'block-id'].join(' ')}>
+                      <Link to={`${routes.blocks.path}/${block.id}`}>
+                        {block.id}
+                      </Link>
+                    </span>
+                    <span className={columnClassNames.height}>{block.height}</span>
+                    <span className={columnClassNames.date}>
+                      <DateTimeFromTimestamp time={block.timestamp * 1000} token="BTC" />
+                    </span>
+                    <span className={columnClassNames.transactions}>
+                      {block.numberOfTransactions}
+                    </span>
+                    <span className={columnClassNames.generator}>
+                      <Link to={`${routes.accounts.pathPrefix}${routes.accounts.path}/${block.generatorAddress}`}>
+                        {block.generatorUsername}
+                      </Link>
+                    </span>
+                    <span className={columnClassNames.amount}>
+                      <LiskAmount val={block.totalAmount} />
             &nbsp;
-                    {`${t('LSK')}`}
-                  </span>
-                  <span className={columnClassNames.forged}>
-                    <LiskAmount val={block.totalForged} />
+                      {`${t('LSK')}`}
```

Comment:
Hey Vit I was thinking that for avoid this fix of spacing maybe we can give some space (margin) to the LiskAmount component and then we can remove the "&nbsp".
What do you think?